snapcraft/storeapi/__init__.py

```diff
@@ -188,6 +189,25 @@ def _refresh_if_necessary(self, func, *args, **kwargs):
             self.conf.save()
             return func(*args, **kwargs)
 
+    def whoami(self):
+        """Return user relevant login information."""
+        account_data = {}
+
+        for k in ('email', 'account_id'):
+            try:
+                value = self.conf.get(k)
+            except KeyError as e:
```

Comment:
Looking at `snapcraft.config.Config()` ([reference](https://github.com/snapcore/snapcraft/blob/master/snapcraft/config.py#L56)), I don't see how this could ever possibly raise a `KeyError`, so this chunk of code seems to do nothing, no?